plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1197,10 +1206,9 @@ elementgui.block.texture_place_bottom_main=Bottom / main
 elementgui.block.texture_place_front=Front
 elementgui.block.texture_place_right=Right
 elementgui.block.texture_place_back=Back
-elementgui.block.block_base_item_texture=Block base (model and behaviour) and optional textures
+elementgui.block.block_base_item_texture=Block base (model and behaviour) and item texture
 elementgui.block.block_base=<html>Block base:<br><small>Most blocks should leave this on default
 elementgui.block.item_texture=<html>Block item texture:<br><small>Optional texture for block in inventory and hand rendering
-elementgui.block.particle_texture=<html>Block particle texture:<br><small>Optional texture for block breaking particles
```

Comment:
Why did you deleted this? You can't make a common key with plant one because there's one word different.

Reply:
I just surely remove it when I tried to fix conflicts.

Reply:
This line is still missing